draft-ietf-cbor-7049bis.md

```diff
@@ -757,10 +757,7 @@ tag and interpret the tagged data item itself.
 
 A tag always applies to the item that is directly followed by it.
 Thus, if tag A is followed by tag B, which is followed by data item C,
-tag A applies to the result of applying tag B on data item C.  That
-is, a tagged item is a data item consisting of a tag and a value.  The
```

Comment:
Why do you remove this?

Reply:
I believe it's redundant with the rest of the section. For example,

> The tag is major type 6, and represents an integer number as indicated by the tag's integer value; the (sole) data item is carried as content data.

above.

Reply:
I don't see how that is redundant.   The removed text provides an order-of-operations concept when multiple tags are chained together on a single data item, which probably shouldn't be lost, might be stated more clearly.   For example, the phrase "while decoding" (or "while encoding)might be helpful (because obviously the order of operations would be reversed while encoding.

For example, there were (hypothetical) tags representing compression (C) and encryption (X) of a single data item, the order of execution the transformations associated with these tags would be important, especially if the code implementing those transformations were separately implemented.    For that case, and with the order-of-operations that works best while decoding would be to decrypt then decompress, so the tags should occur in a CBOR stream would be needed.

Reply:
The text about order of operations is still there. I removed "That is, a tagged item is a data item consisting of a tag and a value.  The content of the tagged item is the data item (the value) that is being tagged."

Reply:
I've reviewed the current text and see what you mean.

Am I right in thinking that a tag technically isn't a item?   If so, I think the remaining language remains a bit confusing, and somehow found the part your removed more convincing that I understood the meaning..

I also think it would be good to make a distinction in this section between the order of application when encoding vs. the order of application while decoding, they would be opposite each other, right?   I does the current description applies to decoding.